package.json

```diff
@@ -13,7 +13,7 @@
     "commit": "cz",
     "nx": "nx",
     "lint-staged": "lint-staged",
-    "generate:provider": "npx hygen provider new --version 0.16.3",
+    "generate:provider": "npx hygen provider new --version 0.22.0",
```

Comment:
Nice 👌🏼 
But I am wondering if there is a way we can automate this version number.

Reply:
Not sure how to read version in package.json 🙂

Reply:
Let's create a linear ticket for that and address it later

Reply:
https://linear.app/novu/issue/NV-3480